recipes/mod_php.rb

```diff
@@ -25,7 +25,7 @@
   Chef::Log.warn("Currently the apache2 cookbook is configured to use the '#{node['apache']['mpm']}' MPM")
 end
 
-case node['platform_family'] # rubocop:disable Style/MultilineIfModifier
```

Comment:
This will break in newer versions of rubocop, the real way to fix this is to refactor as part of #414 .